### PR TITLE
fix: eth deposits through bridge

### PR DESCRIPTION
**Pull request by tynes**
**Description**

Withdrawals don't seem to update balances correctly
for some reason, still need to debug why that is.

### CODE REVIEW

File: packages/sdk/tasks/deposit.ts [hunk 1]
```diff
@@ -108,6 +119,60 @@ task('deposit', 'Deposits WETH9 onto L2.')
       bedrock: true,
     })
 
+    const opBalanceBefore = await signer.provider.getBalance(
+      OptimismPortal.address
+    )
+
+    // Deposit ETH
+    console.log('Depositing ETH through StandardBridge')
+    const ethDeposit = await messenger.depositETH(utils.parseEther('1'))
```
Comment (mslipper) [inline packages/sdk/tasks/deposit.ts:128]: Doesn't this just wait for the transaction on L1 to confirm? Don't we have to call `messenger.waitForMessageReceipt(ethDeposit)`?

File: packages/sdk/tasks/deposit.ts [hunk 2]
```diff
@@ -108,6 +119,60 @@ task('deposit', 'Deposits WETH9 onto L2.')
...
+    if (messageReceipt.status !== 1) {
+      throw new Error('deposit failed')
+    }
+    console.log('Deposit complete')
+
+    const opBalanceAfter = await signer.provider.getBalance(
+      OptimismPortal.address
+    )
+
+    if (!opBalanceBefore.add(utils.parseEther('1')).eq(opBalanceAfter)) {
```
Comment (mslipper) [inline packages/sdk/tasks/deposit.ts:139]: This checks the portal balance, but not the actual balance on L2. Intentional?

**Review by mslipper — CHANGES_REQUESTED** [review]

**Comment by tynes** [comment]
We can merge this and I can follow up with a PR that fixes the bug in the `OptimismPortal`

File: packages/sdk/tasks/deposit.ts (hunk 2 above)
Comment (tynes) [reply packages/sdk/tasks/deposit.ts:139]: This is intentional, the call in the `OptimismPortal` sometimes doesn't actually transfer the ETH

File: packages/sdk/tasks/deposit.ts (hunk 1 above)
Comment (tynes) [reply packages/sdk/tasks/deposit.ts:128]: I didn't think that would wait for the L2 conf

**Review by mslipper — APPROVED** [review]

**Pull request merged**